src/RichContent.js

```diff
@@ -53,7 +53,11 @@ export default class RichContent extends Component {
             const resourceId = this.props.resourceIdsByHrefMap.get(href);
             link.setAttribute("href", `#/resources/${resourceId}`);
           } else {
```

Comment:
Slug already matches what the if condition is checking with `startWith`.
So I would rather use something like this:
```suggestion
          } else if (slug) {
            link.setAttribute("href", `#/resources/${slug}`);
          } else {
            link.setAttribute("href", `#/resources/unavailable`);
          }
```
It's cleaner and has smaller cyclomatic complexity.
Correct me if I'm wrong.

Reply:
Agreed, as we're simply fixing broken links here, this seems like a good options.